src/main/webapp/app/lecture/manage/pdf-preview/pdf-preview-thumbnail-grid/pdf-preview-thumbnail-grid.component.ts

```diff
@@ -61,6 +60,16 @@ export class PdfPreviewThumbnailGridComponent implements OnChanges {
     protected readonly faEyeSlash = faEyeSlash;
     protected readonly faGripLines = faGripLines;
 
+    /**
+     * PDF.js requires a URL pointing to a worker script to process PDFs.
+     * The worker script is included in the package but is required to be served separately as a static asset.
+     * In Artemis the postinstall lifecycle hook (see package.json) copies the script from the package to the
+     * publicly served contents folder.
+     */
```

Comment:
I think it would be good to directly include a link to the repo, and if there is a thread / issue that discusses this topic I would link it with `is required to be served separately as a static asset.`, because that statement might change in the future. 

Besides that the comment looks clear to me 👍 

```suggestion
    /**
     * <a href="https://github.com/mozilla/pdf.js">PDF.js</a> requires a URL pointing to a worker script to process PDFs.
     * The worker script is included in the package but is required to be served separately as a static asset.
     * In Artemis the postinstall lifecycle hook (see package.json) copies the script from the package to the
     * publicly served contents folder.
     */
```

Reply:
done